SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+The Redwood repo is scanned frequently for code and dependency vulnerabilities. Notifications are received by the Redwood Core Team members, who assess risk, prioritize, and determine a remediation plan. Typically this process involves a package update or patch release and is resolved within 24 hours of notification.
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.x.x   | :white_check_mark: |
```

Comment:
I'm not sure I understand this table. What is it for? Is this how it should look?